simtools/camera_efficiency.py

```diff
@@ -524,7 +522,7 @@ def calc_nsb_rate(self):
             * self._telescope_model.reference_data["nsb_reference_value"]["Value"]
             / nsb_integral
         )
-        return nsb_rate, n1_sum
+        return nsb_rate
```

Comment:
And you are sure that n1_sum is not sued when this function is called?

Reply:
But removing it just because we don't know the units is maybe not correct. I think @orelgueta knows the answer here.

Reply:
I removed it because the function was returning a variable that was never use in the code.

Reply:
It's a remnant from the first version of the code which I had locally (there it was used). Either Raul or I copied it but removed its usage. It's safe to remove.